bluemira/equilibria/find.py

```diff
@@ -874,8 +874,10 @@ def _in_plasma(x, z, mask, sep):
         The masking array where 1 denotes inside the zone, and 0 outside
     """
     n, m = x.shape
+    dx = 0.5 * abs(x[0, 1] - x[0, 0])
+    dz = 0.5 * abs(z[1, 0] - z[0, 0])
     for i in range(n):
         for j in range(m):
-            if in_polygon(x[i, j], z[i, j], sep):
+            if in_polygon(x[i, j] + dx, z[i, j] + dz, sep):
```

Comment:
What's the reason for this shift?

Currents and psi should be stored at the same points in the grid, I believe (since current is related to the second derivative of psi). Magnetic fields are offset.

Reply:
honestly I was just trying to get to the bottom of this "discrepancy" we were seeing with FIESTA and was trying this out.

Now that I'm more confident that the discrepancy is due to how the equilibrium constraints / control are being set up differently, I'm less confident this is necessary (it seems to have little effect anyway).

The idea was that when we check for where the plasma current is, that any cell whose centroid is inside the LCFS is registered as carry current.

Reply:
I think I understand the logic - but the cells don't carry current, the nodes do (in this discretisation), and so to lowest order, it should just check if a given node is inside the LCFS. I *think* this is actually adding an error where there wasn't one before...

Reply:
yeah fair